include/chemfiles/formats/PDB.hpp

```diff
@@ -42,13 +43,14 @@ class PDBFormat final: public Format {
     void read_CRYST1(Frame& frame, const std::string& line);
     // Read ATOM and HETATM records
     void read_ATOM(Frame& frame, const std::string& line, bool is_hetatm);
-
+    // Read HELIX records
+    void read_HELIX(const std::string& line);
     // Read CONECT record
     void read_CONECT(Frame& frame, const std::string& line);
 
     std::unique_ptr<TextFile> file_;
-    /// Map of residues, indexed by residue id.
-    std::map<size_t, Residue> residues_;
+    /// Map of residues, indexed by residue id and chainid.
+    std::map<std::pair<char, size_t>, Residue> residues_;
```

Comment:
Why isn't the residue id enough to identify the residue?

Reply:
And they would have the same residue id?

I'll double check what happens with this code when using non-standard PDB (that could have a residue id but no chain id)

Reply:
Yes, the residue IDs all start at 1 for chains A, B, C, D, E, and F. However, the residue sequence for each of them are different!